src/api/spec/features/webui/requests/submissions_spec.rb

```diff
@@ -146,6 +146,28 @@
           expect(page).to have_css('.bg-staging')
         end
       end
+
+      describe 'a request that has a diff comment' do
+        let(:bs_request) do
+          create(:bs_request_with_submit_action,
+                 creator: receiver,
+                 target_project: target_project,
+                 target_package: target_package,
+                 source_project: source_project,
+                 source_package: source_package)
+        end
+        let!(:comment) { create(:comment, commentable: bs_request.bs_request_actions.first, diff_ref: 'diff_0_n1') }
+
+        before do
+          Flipper.enable(:request_show_redesign)
+        end
+
```

Comment:
```suggestion
```
This test is under a `when under the beta program` group of tests, with `beta: true`, on line 107. This is redundant.